pep-0603.rst

```diff
@@ -222,8 +222,9 @@ Implementation
 ==============
 
 The proposed ``frozenmap`` immutable type uses a Hash Array Mapped Trie (HAMT)
-data structure. Functional programming languages, like Clojure, use this novel
-data structure to efficiently implement immutable hash tables.
+data structure. Functional programming languages, like Clojure, use this novel,
+yet often implemented, data structure to efficiently implement immutable hash
```

Comment:
I'm thinking that I shouldn't have used the "novel" word here.  Some people are conservative when they evaluate data structures, so this might make them think about HAMT as something that isn't quite battle tested.

Reply:
Thoughts?

Reply:
Agree. Delete novel.